### ISSUE TITLE
tensorrt 6.0, how to implement  trt.network.get_output(0)[ : , -1, : ] as the network input tensor

### ISSUE THREAD

**Issue by cjsJack** [NVIDIA/TensorRT#502]
## Description

how to  define tensorrt newtork.get_output(0)[:, -1:] as the network input tensor
I use LSTM to train minist data, i define my network` forward() `like this:
`out, (h, c) = self.lstm(x, (h0, c0))`  `# self.lstm -> nn.LSTM()`
`#Decode the hidden state of the last time step`
`out = self.fc( out[ :, -1, : ] )  #self.fc ->  nn.Linear()`

But when i define tensorrt network, how can i implement `network.add_fully_connected(Layer2.get_output(0)[ ;,  -1, : ] )`
It will **error**, how to implement this step
Thanks, hope can help me

## Ubuntu18.04 Python 3.6

**TensorRT 6.0**: 
**GeForce GTX 1660 Ti**: 
**Nvidia Driver Version**: 
**CUDA 10.1**: 
**CUDNN Version**: 
**Operating System + Version**: 
**Python Version 3.6**: 
**TensorFlow Version (if applicable)**: 
**PyTorch Version 1.4**: 
**Baremetal or Container (if container which image + tag)**:

**Comment by ttyio** [comment]
Hello @cjsJack , thanks for question
The operation you are done is a gather operation on the 2nd axis, could you check the `IGatherLayer` in TRT?
C++: https://docs.nvidia.com/deeplearning/tensorrt/api/c_api/classnvinfer1_1_1_i_gather_layer.html
Python: https://docs.nvidia.com/deeplearning/tensorrt/api/python_api/infer/Graph/Network.html#tensorrt.INetworkDefinition.add_gather

**Comment by ttyio** [comment]
I will close this since no response for more than 3 weeks, please reopen if you still have question, thanks!
